CRM/Price/BAO/LineItem.php

```diff
@@ -333,6 +333,11 @@ public static function getLineItems($entityId, $entity = 'participant', $isQuick
    *   this is
    *                          lineItem array)
    * @param string $amount_override
+   *   Amount override must be in format 1000.00 - ie no thousand separator & if
+   *   a decimal point is used it should be a decimal
```

Comment:
On further investigation I have ensured that in codebase the $amount_override carry the value from $params['partial_payment_total'] which is always in 1000.00 format as the value is formatted through ```CRM_Utils_Rule::cleanMoney($params['partial_payment_total'])```

These are two important points where ```partial_payment_total``` parameter is assigned:
https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/BAO/Contribution.php#L3191
https://github.com/civicrm/civicrm-core/blob/master/CRM/Event/Form/Participant.php#L1431L1442